pyproject.toml

```diff
@@ -14,3 +14,9 @@ minversion = "6.2"
 testpaths = [
     "tests",
 ]
+
+[tool.black]
+line-length = 500
+
+[tool.isort]
+line_length = 500
```

Comment:
I'm not sure I'd be happy with a 500 line length file. Other tools in the ecosystem are already enforcing 120 char line length

Reply:
The 500 length does not affect the enforcement of 120 length for flake8 when delivering code. This was simply to assist in the code generation process, where the post processing in `scripts/oscal_normalize` was getting confused.

Reply:
Makes sense to me